src/test/java/chess/position/DirectionTest.java

```diff
@@ -0,0 +1,91 @@
+package chess.position;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class DirectionTest {
+
+    Position source = Position.of(File.D, Rank.FOUR);
```

Comment:
```suggestion
    private Position source = Position.of(File.D, Rank.FOUR);
```
외부 사용을 막아주어야할 것 같네요 :)

Reply:
테스트코드에서도 변수를 접근제어자로 막아주어야 하나요?!

Reply:
호기는 막았을 때와 안막았을 때 각각 어떤 차이가 있다고 생각하시나요?

Reply:
같은 패키지의 다른 테스트에서 변수에 접근을 할 수가 있겠네요.

테스트가 순서대로 실행이 되는 것이 아니기 때문에 다른 테스트에서 변수를 만졌을 경우 테스트가 깨질 수 있을 것 같아요..!

고민해보지 못한 부분을 알려주셔서 감사합니다!